.changeset/tricky-taxis-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effect/cli": minor
```

Comment:
I don't think so, because it fundamentally changes the way arguments are handled internally. Previously, you had to strip the executable manually when passing arguments in (`process.argv.slice(2)`). Now, to allow the optionality of the executable being provided, that is done internally for you.

Reply:
Ah yes, forgot about that.